grpc/execution/server_test.go

```diff
@@ -463,6 +465,7 @@ func TestExecutionServiceServerV1Alpha2_ExecuteBlockAndUpdateCommitment(t *testi
 	require.NotNil(t, stateDb, "State db is nil")
 	chainDestinationAddressBalanceAfter := stateDb.GetBalance(chainDestinationAddress)
 
-	balanceDiff := new(big.Int).Sub(chainDestinationAddressBalanceAfter, chainDestinationAddressBalanceBefore)
-	require.True(t, balanceDiff.Cmp(big.NewInt(1000000000000000000)) == 0, "Chain destination address balance is not correct")
+	balanceDiff := new(uint256.Int).Sub(chainDestinationAddressBalanceAfter, chainDestinationAddressBalanceBefore)
+	fmt.Printf("Balance diff: %s\n", balanceDiff.String())
+	require.True(t, balanceDiff.Cmp(uint256.NewInt(1000000000000000000)) == 0, "Chain destination address balance is not correct")
```

Comment:
examples of some changes to uint256.Int. This is because the stateDB is now returning only uint256s.